pkg/cmd/start/start.go

```diff
@@ -104,95 +104,10 @@ func NewGatherAndUpload() *cobra.Command {
 		Short: "Runs the data gathering as job, uploads the data, waits for Insights analysis report and ends",
 		Run:   runGatherAndUpload(operator, cfg),
 	}
+	cmd.PersistentFlags().String("storagePath", "", "Path to store the gathered data")
```

Comment:
Why do we need this flag please? My thinking was that we can read this from API and if it's empty then use the default value or am I missing something?

Reply:
I was using this flag to propagate the value from the API and IO to the container where we are executing the command. In the `runGatherAndUpload` function I am then just checking if the flag was provided, if so then I will use it, otherwise it uses default path. I think the main reason was that the command is used for both periodic and on-demand gathering jobs and if the path was changed in the `DataGather` CRD then I needed to use it in a job command. But there might be an easier way to achieve this that I haven't found...

Reply:
I have reworked this to not use the flag. I did not notice that we are passing the `DataGather` name to the job as a environment variable and then getting the CRD in a job... Thanks for pointing this out!